packages/cardano-services-client/test/util.ts

```diff
@@ -22,3 +22,79 @@ export const healthCheckResponseWithState = {
   },
   ok: true
 };
+
+export const getAliceHandleProviderResponse = {
+  handle: 'alice',
+  hasDatum: false,
+  policyId: Cardano.PolicyId('50fdcdbfa3154db86a87e4b5697ae30d272e0bbcfa8122efd3e301cb'),
+  resolvedAddresses: {
+    cardano:
+      'addr_test1qqk4sr4f7vtqzd2w90d5nfu3n59jhhpawyphnek2y7er02nkrezryq3ydtmkg0e7e2jvzg443h0ffzfwd09wpcxy2fuqmcnecd'
+  },
+  resolvedAt: {
+    hash: Cardano.BlockId('10d64cc11e9b20e15b6c46aa7b1fed11246f437e62225655a30ea47bf8cc22d0'),
+    slot: Cardano.Slot(37_834_496)
+  }
+};
+
+export const getBobHandleProviderResponse = {
+  handle: 'bob',
+  hasDatum: false,
+  policyId: Cardano.PolicyId('50fdcdbfa3154db86a87e4b5697ae30d272e0bbcfa8122efd3e301cb'),
+  resolvedAddresses: {
+    cardano:
+      'addr_test1qzrljm7nskakjydxlr450ktsj08zuw6aktvgfkmmyw9semrkrezryq3ydtmkg0e7e2jvzg443h0ffzfwd09wpcxy2fuql9tk0g'
+  },
+  resolvedAt: {
+    hash: Cardano.BlockId('10d64cc11e9b20e15b6c46aa7b1fed11246f437e62225655a30ea47bf8cc22d0'),
+    slot: Cardano.Slot(37_834_496)
+  }
+};
+
+export const getAliceHandleAPIResponse = {
+  background: 'zrljm7nskakjydxlr450ktsj08zuw6aktvgfkmmyw9semrkrezryq3yd',
+  characters: 'rljm7n/23455',
+  created_slot_number: 33,
+  default_in_wallet: 'alice_default_hndle',
+  hasDatum: false,
+  hex: '0f3abbc8fc19c2e61bab6059bf8a466e6e754833a08a62a6c56fe0e78f19d9d5',
+  holder_address:
+    'addr_test1qzrljm7nskakjydxlr450ktsj08zuw6aktvgfkmmyw9semrkrezryq3ydtmkg0e7e2jvzg443h0ffzfwd09wpcxy2fuql2erwd',
```

Comment:
```suggestion
    'stake1uyehkck0lajq8gr28t9uxnuvgcqrc6070x3k9r8048z8y5gh6ffgw',
```
Holder address is a _stake address_.

https://api.handle.me/swagger/#/handles/get_handles__handle_